cirq/testing/circuit_compare.py

```diff
@@ -364,6 +366,9 @@ def _infer_qubit_count(val: Any) -> int:
         return 3
     if isinstance(val, ops.ControlledGate):
         return 1 + _infer_qubit_count(val.sub_gate)
+    if protocols.has_unitary(val):
+        return protocols.unitary(val).shape[0].bit_length() - 1
```

Comment:
Use `default` argument of unitary instead of calling `has` then `unitary`.

Reply:
I realized this is actually unnecessary because because the function that uses this helper method already checks that case.